### ISSUE TITLE
View of document storage

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#5]
## User story
1. As a user
2. I want to be able to view all the documents that have been uploaded
3. So that I can get an overview of what is stored
---Add mockup

## Acceptance criteria
* User can see all the files that have been uploaded successfully and are currently stored in the backend.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
